app/models/group.rb

```diff
@@ -0,0 +1,17 @@
+class Group < ApplicationRecord
+  belongs_to :user
+  has_many :group_moves, dependent: :destroy
+  has_many :moves, through: :group_moves
+
+  validates :name, presence: true
+  validates :icon, presence: true
```

Comment:
- [ ] You've done an excellent job on this project, congrats! However, I would suggest validating the type of data for :name and :amount, for example, you could say :amount should be an integer greater than cero.

Reply:
thanks, @smunozmo I implemented it now